src/Infrastructure/User/Query/Projections/UserView.php

```diff
@@ -8,18 +8,18 @@
 use App\Domain\User\ValueObject\Auth\Credentials;
 use App\Domain\User\ValueObject\Auth\HashedPassword;
 use App\Domain\User\ValueObject\Email;
-use Broadway\ReadModel\SerializableReadModel;
-use Broadway\Serializer\Serializable;
+use Messenger\Event\EventInterface;
+use Messenger\Projection\ReadModelInterface;
 use Ramsey\Uuid\Uuid;
 use Ramsey\Uuid\UuidInterface;
 
-class UserView implements SerializableReadModel
+class UserView implements ReadModelInterface
 {
     /**
      * @throws \App\Domain\Shared\Exception\DateTimeException
      * @throws \Assert\AssertionFailedException
      */
-    public static function fromSerializable(Serializable $event): self
+    public static function fromSerializable(EventInterface $event): self
```

Comment:
EventInterface doesn't match the needs here. Serializable it's the projection itself not an event